pyspecProcScripts/first_level/fake_data.py

```diff
@@ -0,0 +1,95 @@
+from .fl_dummy import fl_dummy
+from sympy import Symbol as sympy_symbol
+from sympy import lambdify
+from pyspecdata import nddata, ndshape
+import numpy as np
+from numpy.random import normal
+def fake_data(
+        expression,
+        axis_coords,
+        signal_pathway,
+        direct = 't2',
+        SD_sigma = [0.05,0.003],
```

Comment:
will this be true for most data sets we create? I'm still iffy on how sigma/gaussians contribute to the fake data signal. Do you have references you can point me to to better connect these ideas?

Reply:
We are generating a random list of frequency offsets to model the field drift. These Gaussians are used to give those numbers a non-uniform spectral density.